tools/generate_opcodes_all.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env sh
+
+# This script generates internal/vm/opcodes/all.go.
+
+set -e
+
+temp_file=$(mktemp)
+
+(
+	# shellcheck disable=SC2016
+	echo '// Generated by generate_opcodes_all.sh. Run `make fix` to update. DO NOT EDIT.'
```

Comment:
You could consider replacing the repeated echo statements with a [heredoc](https://linuxize.com/post/bash-heredoc). That said, the current version is quite readable, totally fine to leave as-is if you prefer.

Reply:
I prefer the echo statements so I'll leave it as-is 🙂